### PR TITLE
Fixes outdated tips

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
Cloning? Charcoal? Antitoxin? What're you talkin' about old man, the future is now *snaps fingers*

## Why It's Good For The Game
prevents confusion

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -41,17 +41,19 @@ As the Captain, always take the nuclear disk and pinpointer with you every shift
...
+As a Medical Doctor, corpses with the "...and their soul has departed" description no longer have a ghost attached to them and can't be revived.
 As a Medical Doctor, treating plasmamen is not impossible! Salbutamol stops them from suffocating and showers stop them from burning alive. You can even perform surgery on them by doing the procedure on a roller bed under a shower.
 As a Medical Doctor, you can point your penlight at people to create a medical hologram. This lets them know that you're coming to treat them.
 As a Medical Doctor, you can extract implants by holding an empty implant case in your offhand while performing the extraction step.
-As a Medical Doctor, clone scanning people will implant them with a health tracker that displays their vitals in the clone records. Useful to check on crew members that didn't activate suit sensors!
+As a Medical Doctor, you can deal with patients who have absurd amounts of wounds by putting them in cryo. This will slowly treat all of their wounds simultaneously, but is much slower than direct treatment.
```
Comment (ATH1909) [inline strings/tips.txt:53]: please specify that you should NOT try to put a corpse in cryo

File: strings/tips.txt [hunk 2]
```diff
@@ -41,17 +41,19 @@ As the Captain, always take the nuclear disk and pinpointer with you every shift
...
+As a Medical Doctor, Critical Slash wounds are one of the most dangerous conditions someone can have. Apply gauze, epipens, sutures, cauteries, whatever you can, as soon as possible!
```
Comment (ATH1909) [inline strings/tips.txt:54]: *dabs in coagulant+gauze*

File: strings/tips.txt [hunk 3]
```diff
@@ -172,7 +174,7 @@ As a Shaft Miner, every monster on Lavaland has a pattern you can exploit to min
 As a Shaft Miner, you can harvest goliath plates from goliaths and upgrade your explorer's suit, mining hardsuits as well as Firefighter APLUs with them, greatly reducing incoming melee damage.
 As a Shaft Miner, always have a GPS on you, so a fellow miner or cyborg can come to save you if you die.
 As a Shaft Miner, you can craft a variety of equipment from the local fauna. Bone axes, lava boats and ash drake armour are just a few of them!
-As a Traitor, the cryptographic sequencer (emag) can not only open doors, but also lockers, crates, APCs and more. It can hack cyborgs, and even cause bots to go berserk. Use it on the right machines, and you can even order more traitor gear or contact the Syndicate. Experiment!
+As a Traitor, the cryptographic sequencer (emag) can not only open lockers, crates, APCs and more. It can also do things like hack cyborgs, and even cause bots to go berserk. Use it on the right machines, and you can even contact the Syndicate. Experiment!
```
Comment (ATH1909) [inline strings/tips.txt:177]: that first sentence is a bit oddly phrased

**Comment by Pugie** [comment]
Whatever you do, don't remove the flashbanging blob tip

**Pull request merged**